internal/display/organizations.go

```diff
@@ -78,17 +78,17 @@ func (r *Renderer) OrganizationUpdate(organization *management.Organization) {
 	r.Result(makeOrganizationView(organization, r.MessageWriter))
 }
 
-func makeOrganizationView(organization *management.Organization, w io.Writer) *organizationView {
+func makeOrganizationView(organization *management.Organization, _ io.Writer) *organizationView {
```

Comment:
Can we remove the `_ io.Writer` parameter?

Reply:
Fixed in https://github.com/auth0/auth0-cli/pull/489/commits/802fb15094837e7be3b28f8866430d2f9d5f887f